### DISCUSSION TITLE
Installing/Running Lighting Node on Windows

### DISCUSSION THREAD

**Discussion by afjol97** [lightningnetwork/lnd#8227] [Q&A]
Hello, Hope you are well.

So i've downloaded Bitcoin Node which is synced and ready (Still running). I downloaded LND latest beta for windows (ARM64+386 Version installed. just in case. Now i am confused on the next step. 

I extracted the folder and then I right click the lnd.exe file and then open but the program opens and then closes
i right click the lncli.exe and then open but i get the same result.

What am i doing wrong? 

Would it be easier to change OS so this is done smoothly? If so would i need to download Bitcoin Node again? i also i merged two hdd together to give me 2tb of space which i have bitcoin node installed in and i also have an m.2 ssd just for my OS. 

I'm sorry if my request for help looks stupid. I will do my best to educate myself and understand further but at the moment i am very confused and stuck. All help is appreciated. 

Edit: I have a gaming pc which is currently on Windows OS that i want to use for Bitcoin node and Lighting Node. I know i can buy a RaspberryPi and things are more simple but id rather not spend money. I do have funds ready for the Lighting Node.

**Comment by guggero** [comment]
Both `lnd.exe` and `lncli.exe` are command line tools. So you need to use a terminal and then type in the commands.
This article I found through Google is really old, but maybe it has some clues for you: https://medium.com/@martinschrer/installing-a-mainnet-lnd-node-on-windows-and-accessing-it-on-an-iphone-da1e63a613cb
